### ISSUE TITLE
Unprotect Object

### ISSUE THREAD

**Issue by jidelaf** [bseltz-cohesity/scripts#159]
Hi Brian,
In powershell script script [unprotectObjects.ps1](https://github.com/bseltz-cohesity/scripts/tree/master/powershell/unprotectObjects)

I'm trying to exclude a Netapp volume form a protection job.

```
./unprotectObjects.ps1 -clusterName MYCLUESTERNAME -objectName flexclone_volumename -jobName NAS_NetappSVMname
Connected!
    NOTFOUND: flexclone_volumename (not found in any protection jobs)

Output saved to log-unprotectObjects-2024-06-06.txt
```

When checking COhesity Web gui I can verify that the backup gropu name and volume names are good.
Should this script work for netapp backup group?
The Netapp source is registered at SVM level.
![unprotectnetapp](https://github.com/bseltz-cohesity/scripts/assets/170001374/81edcba9-b464-47cc-9db2-3aae723cb004)

**Comment by bseltz-cohesity** [comment]
Keep in mind that the script can only work if the volume is explicitly selected in the protection group. If the NetApp cluster/SVM is autoprotected, then the volume can not be unselected.

**Comment by jidelaf** [comment]
And you are right :) 
We use autoprotect at SVM level.

Do you know if there is any alternative powershell script to exclude Netapp volumes?

I see we can do it form the GUI in the screenshot we have one excluded already
![unprotectnetapp2](https://github.com/bseltz-cohesity/scripts/assets/170001374/2d0d6aec-655c-4992-8c7f-cb5922483352)

When decommissioning a Netapp volume, our Netapp admins mark the volumes offline for several days before they actually delete it.
During those days and until the volume gets deleted, we receive alerts such as "[1-5-1] No supported backup type for SVMname:volumename. Currently we only support NFSv3, NFSv4.1 or SMB(1-3) backup, so please check its supported NAS protocols and corresponding configurations"

So ideally we would like to have some script for them to run whenever they offline a volume that automtically exclude the volumes

**Comment by bseltz-cohesity** [comment]
Yes, you can use the protectNetapp.ps1 script: https://github.com/cohesity/community-automation-samples/tree/main/powershell/protectNetapp 

```
./protectNetapp.ps1 -clusterName mycluster -jobName myjob -netappSource My-NetApp -vexString vol1
```

**Comment by jidelaf** [comment]
I have tested it with protectNetapp.ps1 and it works like a charm 
Thank you!

The only bad part is that it doesn't give you any output about whether it was excluded OK or not, if it was excluded aleady or what are the current exclusions.
So we can't validate it was applied properly. (Only checking the Web gui)
Is there any way to check current exclusions for a Netapp backup job?

**Comment by bseltz-cohesity** [comment]
I have updated the script to report what it is protecting/excluding. https://github.com/cohesity/community-automation-samples/tree/main/powershell/protectNetapp
